assets/i18n/en.json

```diff
@@ -14,10 +14,14 @@
     "I18N_ABOUT_PAGE_ABOUT_TAB_PARAGRAPH_8": "Whether you're a K-12 educator, a graduate student, or an individual who's passionate about a specific subject and wants to share your knowledge, Oppia welcomes you. Join the community and start exploring with us.",
     "I18N_ABOUT_PAGE_ABOUT_TAB_SHARE": "Publish &amp; Share",
     "I18N_ABOUT_PAGE_ABOUT_TAB_SHARE_TEXT": "your creations with the community.",
+    "I18N_ABOUT_PAGE_CREATE_LESSON_CONTENT": "With Oppia's content creation system, you can easily customize the lesson on topics you are passionate about.",
     "I18N_ABOUT_PAGE_CREDITS_TAB_HEADING": "Credits",
     "I18N_ABOUT_PAGE_CREDITS_TAB_TEXT": "Oppia's contributors come from all over the world &mdash; many of us are students, recent students and teachers. We'd like to thank the following contributors who've helped to build the platform. If you'd like to help, <a href=\"/contact\">here's how you can get involved</a>!",
     "I18N_ABOUT_PAGE_CREDITS_TAB_TEXT_BOTTOM": "The Oppia development team is also grateful for the feedback, ideas, help and suggestions by <[listOfNames]>.",
     "I18N_ABOUT_PAGE_CREDITS_THANK_TRANSLATEWIKI": "We would also like to thank <a href=\"https://translatewiki.net/wiki/Translating:Oppia\">translatewiki.net</a> for providing crowdsourced translations.",
+    "I18N_ABOUT_PAGE_EASILY_CREATE_LESSON": "Easily Create Lessons",
+    "I18N_ABOUT_PAGE_EXPLORE_LESSONS": "Explore Lessons Made by the Community",
+    "I18N_ABOUT_PAGE_EXPLORE_LESSONS_CONTENT": "Educators and community members around the world use Oppia's lesson creation platform as a way to create and share lessons.You can find over 20,000 lessons for 17 different subjects in our Exploration library, and maybe you'll be inspired to create you own!",
```

Comment:
Add space before "You"

you own --> your own

Reply:
Done